openedx_external_enrollments/external_enrollments/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""External enrollments method file."""
+import logging
+
+from openedx_external_enrollments.edxapp_wrapper.get_site_configuration import configuration_helpers
+from openedx_external_enrollments.factory import ExternalEnrollmentFactory
+
+LOG = logging.getLogger(__name__)
+
+
+def execute_external_enrollment(data, course):
+    """
+    Execute an enrollment for the given data and course.
+
+    Args:
+        data: dict with the enrollment data.
+        course: instance of CourseDescriptor.
+    """
+    try:
+        controller = course.other_course_settings.get('external_platform_target', '')
+    except AttributeError:
+        LOG.error('Course [%s] not configured as external.', str(course.id))
```

Comment:
Will this appear for all non-external courses? If so, we should avoid it or hide it behind a site configuration because this will be called in any post enrollment save across the platform, not just for mylearningx courses. @andrey-canon